dist/npm/package.json

```diff
@@ -1,7 +1,28 @@
 {
   "name": "quick-lint-js",
   "description": "Find bugs in your JavaScript code",
-  "version": "0.1.1-a",
+  "version": "0.1.1-b",
+  "keywords": [
+    "quick",
+    "lint",
+    "js",
+    "syntax",
+    "errors",
+    "bugs",
+    "fast",
+    "static-analysis",
+    "code-quality",
+    "feedback",
+    "javascript",
+    "linter",
+    "linting",
+    "strager",
+    "lint-quick-js",
+    "js-quick-lint",
+    "js-lint-quick",
+    "quick-js-lint",
+    "lint-js-quick"
```

Comment:
Nit: Alphabetically sort these, please. strageP

Reply:
I don't think alphabetical sort makes sense in this instance. I ranked them on the relevance of what I thought would be most likely searched for, secondly on the descriptiveness of what quick-lint does.